Xenakios/TakeRenaming.cpp

```diff
@@ -114,149 +115,84 @@ void AddToRenameLog(string &oldfilename,string &newfilename)
 #endif
 }
 
-void DoRenameSourceFileDialog666(COMMAND_T* ct)
+// ct->user==0: rename source only, ==1: rename source and take
+void DoRenameTakeAndSourceFileDialog(COMMAND_T* ct)
 {
 	vector<MediaItem_Take*> thetakes;
 	vector<MediaItem_Take*> alltakes;
 	XenGetProjectTakes(alltakes,false,false);
 	XenGetProjectTakes(thetakes,true,true);
 	if (thetakes.size()==0) return;
-	g_renameparams.takesToRename=(int)thetakes.size();
-	g_renameparams.mode=1;
-	bool bChanges = false;
-	for (int i=0;i<(int)thetakes.size();i++)
+	g_renameparams.takesToRename=thetakes.size();
+	g_renameparams.mode=2;
```

Comment:
`mode` should be either 1 or 2 depending on `ct->user` to give the correct title to the dialog. It's always "Rename take and source file" now.